py3status/modules/air_quality.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+"""
+Display air quality polluting in a given location.
+
+An air quality index (AQI) is a number used by government agencies to communicate
+to the public how polluted the air currently is or how polluted it is forecast to
+become. As the AQI increases, an increasingly large percentage of the population
+is likely to experience increasingly severe adverse health effects. Different
+countries have their own air quality indices, corresponding to different national
+air quality standards.
+
+Configuration parameters:
+    auth_token: Required. Personal token. http://aqicn.org (default 'demo')
+    cache_timeout: refresh interval for this module. A message from the site:
+        The default quota is max 1000 (one thousand) requests per minute (~16RPS)
+        and with burst up to 60 requests. Read more: http://aqicn.org/api/
+        (default 3600)
+    format: display format for this module (default 'AQI: {aqi} {category}')
+    location: location/uid to query. To search for nearby stations in Kraków,
+        use `curl http://api.waqi.info/search/?token=YOUR_TOKEN&keyword=kraków`
+        We recommend you to use uid instead of name in location, eg "@8691"
+        (default 'Shanghai')
+
+Format placeholders:
+    {aqi} air quality index
+    {category} health risk category
+    {location} location/uid
+    {something} output from "something" parameter in `iaqi` array.
```

Comment:
I think we should remove `{something}` as it is not a placeholder.

maybe a comment like 
```A station may have their own additional set of custom parameters such as {pm25}, {pm10}, etc. You can find these by using `curl http://api.waqi.info/feed/LOCATION?token=YOUR_TOKEN` and look for them```